t/rose-app-run/06-namelist.t

```diff
@@ -177,22 +177,28 @@ test_teardown
 TEST_KEY=$TEST_KEY_BASE-install-only
 test_setup
 run_pass "$TEST_KEY" rose app-run --config=../config -i --debug
-file_cmp "$TEST_KEY.out" "$TEST_KEY.out" <<__CONTENT__
+#sort "$TEST_KEY.out" > "$TEST_KEY.sorted.out"
+python3 -c "import re, sys
```

Comment:
This test previously seemed sensitive to the output order, which didn't actually seem to be an set in the script being tested.